spec.go

```diff
@@ -131,6 +131,10 @@ type Artifacts struct {
 	Directories *CreateArtifactDirectories `yaml:"createDirectories,omitempty" json:"createDirectories,omitempty"`
 	// ConfigFiles is a list of files that should be marked as config files in the package.
 	ConfigFiles map[string]ArtifactConfig `yaml:"configFiles,omitempty" json:"configFiles,omitempty"`
+	// DocFiles is a list of doc files included in the package
```

Comment:
These doc strings are not correct anymore 🙈

Reply:
```suggestion
	// Docs is a list of doc files included in the package
```

Reply:
LOL sorry